packages/api/src/common/error/middleware/errorHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { Middleware } from "koa";
+import { HttpError } from "../classes/httpError";
+
+export default (): Middleware => async (ctx, next) => {
+    try {
+        await next();
+    } catch (err) {
+        if (err instanceof HttpError) {
+            ctx.status = err.status;
+            ctx.body = {
+                status: err.status,
+                message: err.reason
+            };
+
+            return;
+        }
+
+        console.error(err);
```

Comment:
Did you lave this here intentionally?

Reply:
I guess BG added it. I don't remember adding this.